scripts/globals/mobskills/bomb_toss.lua

```diff
@@ -1,5 +1,5 @@
 -----------------------------------
--- Bomb Toss
+-- Bomb Toss ("Suicide Fail")
```

Comment:
This one says "suicide fail" in `bomb_toss.lua` but there's already a bomb_toss_suicide.lua?

Reply:
It means it "failed to suicide" - I used Jimmayus wording off the sheet I linked. I can make it more understandable

Reply:
oh that makes sense in context